lib/vfscore/vfs.h

```diff
@@ -69,89 +69,728 @@ extern int vfs_debug;
  * per task data
  */
 struct task {
-	char 	    t_cwd[PATH_MAX];	/* current working directory */
-	struct vfscore_file *t_cwdfp;		/* directory for cwd */
+	char t_cwd[PATH_MAX];		  /* current working directory */
+	struct vfscore_file *t_cwdfp; /* directory for cwd */
 };
 
-int	 sys_open(char *path, int flags, mode_t mode, struct vfscore_file **fp);
-int	 sys_read(struct vfscore_file *fp, const struct iovec *iov, size_t niov,
+/**
+ * Opens the file specified by path
+ *
+ * @param path
+ *	Path to the file to be opened
+ * @param flags
+ *	Argument flags regarding how the file should be opened
```

Comment:
```suggestion
 *	Flags regarding how the file should be opened
```
I would give a couple of flags examples here.